client/resource_group/controller/controller.go

```diff
@@ -31,7 +32,7 @@ import (
 )
 
 const (
-	requestUnitConfigPath  = "resource_group/ru_config"
+	controllerConfigPath   = "resource_group/control"
```

Comment:
Maybe a little bit more specific would be better.

```suggestion
	controllerConfigPath   = "resource_group/controller_config"
```

Reply:
Do we need the suffix?  Other config key path don't have "_config".